src/components/List.js

```diff
@@ -79,6 +80,53 @@ const List = ({ token }) => {
     return daysUntilNextPurchase > 0 ? daysUntilNextPurchase : 0;
   };
 
+  const sortedList = docs.sort((a, b) => {
+    const aDaysUntilNextPurchase = getDaysUntilNextPurchase(a);
+    const bDaysUntilNextPurchase = getDaysUntilNextPurchase(b);
+
+    if (!isInactive(a) && isInactive(b)) {
+      return -1;
+    }
+    if (isInactive(a) && !isInactive(b)) {
+      return 1;
+    }
+
+    if (aDaysUntilNextPurchase < bDaysUntilNextPurchase) {
+      return -1;
+    }
+    if (aDaysUntilNextPurchase > bDaysUntilNextPurchase) {
+      return 1;
+    }
+    if (aDaysUntilNextPurchase === bDaysUntilNextPurchase) {
+      if (a.itemName < b.itemName) {
+        return -1;
+      } else {
+        return 1;
+      }
+    }
+    return 0;
+  });
+
+  function isInactive(item) {
```

Comment:
By adding this helper function `isInactive` we can use the 'inactive' property as a boolean rather than just a string which we are able to use in the sorting function conditional logic to sort the inactive items to the end of the list.